ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-postsubmits.yaml

```diff
@@ -210,3 +210,56 @@ postsubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^master$
+    cluster: build01
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci.openshift.io/generator: prowgen
+    max_concurrency: 1
+    name: branch-ci-openshift-assisted-service-master-unit-test-postsubmit
+    skip_if_only_changed: ^docs/|^sites/|\.md$|^(LICENSE|OWNERS|OWNER_ALIASES)$
```

Comment:
not critical, but maybe we can always run it in this case of post-submit
it will not be a great waste of resources in my opinion

Reply:
makes sense, I'll change it :+1: